### ISSUE TITLE
hyper-parameter

### ISSUE THREAD

**Issue by huanmofazong** [aigagror/ml_aesthetics#19]
Thank you for providing the code. I would like to ask, according to the learning rate and batch_size you provided, how many epoches are needed to get the effect?
I used the learning rate you gave, but modified batch_size=32 and trained 20 epoches, but there is still a certain gap.
